### DISCUSSION TITLE
使用doris官网的SQLServer同步执行成功，但是uniqueidentifier类型不支持

### DISCUSSION THREAD

**Discussion by AndLlk** [apache/doris#28306] [A - General / Q&A]
![企业微信截图_17012400514591](https://github.com/apache/doris/assets/60912041/8ab2c5f4-10ac-4c32-9501-8e6399e3f2a0)
类型报错 uniqueidentifier,varbinary 
官方能支持一下这几个类型吗，有啥好办法

**Comment by AndLlk** [comment]
![image](https://github.com/apache/doris/assets/60912041/d448a508-ea72-49f0-99cb-898502cbd4ab)
添加的jar包，测试表 实时同步成功

**Reply by LemonLiTree** [reply]
可以在这里记录一个issue
https://github.com/apache/doris-flink-connector/issues

**Reply by AndLlk** [reply]
好的
